bioframe/io/resources.py

```diff
@@ -90,10 +90,72 @@ def fetch_chromsizes(
         raise ValueError("Unknown provider '{}'".format(provider))
 
 
+def _centers_from_cytoband(cyb):
```

Comment:
I used "center" instead of midpoint because the division isn't necessarily at the half-way point between the extents of the `acen` bands. Maybe "origin" is a better term.